documentation/guide-api-first.asciidoc

```diff
@@ -75,34 +65,34 @@ If we already have our backend maven project, using the maven plugin will be eas
+.Generated sources by OpenAPI generator
+image::images/apifirst-generated-sources.png["Generated sources by OpenAPI generator",scaledwidth="80%",align="center",link="https://devonfw.com/website/pages/docs/images/apifirst-generated-sources.png"]
 
 
-Depending on the configuration of your codegen, you might need to add some additional dependencies to your `pom.xml` for example for bean validation or Jackson annotations.
+Depending on the configuration of your code generator, you might need to add some additional dependencies to your `pom.xml` for example for bean validation or Jackson annotations.
 
-The codegen has many more configuration options that our outside the scope of this guide and you can find full documentation of the codegen plugin in the https://github.com/OpenAPITools/openapi-generator/tree/master/modules/openapi-generator-maven-plugin[projects repo]
+The OpenAPI generator has many more configuration options that our outside the scope of this guide and you can find full documentation of the codegen plugin in the https://github.com/OpenAPITools/openapi-generator/tree/master/modules/openapi-generator-maven-plugin[projects GitHub repository].
 
 
 === Generate once vs generate always
 
 Depending on our needs, we may either want to generate the interfaces and models once and afterwards copy them to our project as general source code files, or treat them as immutable generated assets, that we generate anew with every build.
-Both scenarios have their pros and cons and you'll need to find out what best suits your project. In the example above we use a profile with `activeByDefault=true` which will cause the generator to run with every build. The generated files will be included as sources in our project, so we can import them in any other java class without issues.
+Both scenarios have their pros and cons, and you'll need to find out what best suits your project. In the example above we use a profile with `activeByDefault=true` which will cause the generator to run with every build. The generated files will be included as sources in our project, so we can import them in any other java class without issues.
 
-In case you want to generate your API resorces only once and afterwards version them in SCM, simply run the generator, outputing to some temp location, afterwards copy them to `src/main/java` and your are done. Be careful if you manualy modify the generated files afterwards and you want to re-generate them after API schema update, you will loose any manual changes. 
+In case you want to generate your API resources only once and afterwards version them in SCM, simply run the generator, outputing to some temp location, afterwards copy them to `src/main/java` and your are done. Be careful if you manualy modify the generated files afterwards and you want to re-generate them after API schema update, you will loose any manual changes. 
```

Comment:
In case you **only** want to generate your API resources **once** and version them **afterwards** in SCM, simply run the generator, **outputting** to some temp (temporary?) location, then, copy them to `src/main/java` and (you are OR you're) done. Be careful if you **manually** modify the generated files afterwards and re-generate them after **an** API schema update, you will **lose** any manual changes.